### PR TITLE
Create JS function to generate to do list lines from object list

### PR DESCRIPTION
**Pull request by leonardodiasb**
Created JS function to generate, from an object list, the lines of To-Do activities in the HTML.
Created a box to fit the To-Do activities and style in CSS for it. 
Used Webpack.

### CODE REVIEW

**Review by ZawadiSumba66 — CHANGES_REQUESTED** [review]
### Hi @leonardodiasb 👋 , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request 🛑 

You have submitted a project, that has one of the linter checks failing. I also realized that your linter configuration is incorrect since your project cannot find the necessary linter files to check.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: webpack.config.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  ],
+  module: {
+    rules: [
+      {
+        test: /\.css$/i,
+        use: ['style-loader', 'css-loader'],
+      },
+    ],
+  },
+};
```
Comment (teekaytech) [inline webpack.config.js]: - [ ] You need to configure the basic commands for building Webpack in development and production mode using the `package.json` file. I believe you've done something similar in your Webpack exercise. We should be able to set up and run Webpack devserver with no errors, using the configuration. At the moment, package.json is completely missing in your project. Kindly fix this.

File: README.md [hunk 2]
```diff
@@ -1 +1,27 @@
...
+
+- GitHub: [@leonardodiasb](https://github.com/leonardodiasb)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+## Show your support
+
+Give a ⭐️ if you like this project!
```
Comment (teekaytech) [inline README.md]: **OPTIONAL**

- [ ] Kindly update your Readme file
- I strongly suggest you add the `Getting Started` section, where you state the instructions and code samples that will be needed to clone your project, install its dependencies, and starting the project on a dev environment.
- The live demo link is broken: if you don't have a live demo yet, you can remove this link. It is not a good practice to have broken link(s) in your Readme.
- You may also add a screenshot of the project to your Readme.

**Review by teekaytech — CHANGES_REQUESTED** [review]
## Status: Changes Required :warning: :vertical_traffic_light: 

### Hello @leonardodiasb :wave:

_Tis milestone is almost complete as most of the tasks have been implemented :handshake:_

- Your PR title, summary and commit messages look okay :ok_hand: 
- All linter checks are running with no errors :100: 

__However, you need to make some required changes before moving on to the next assignment.__
> Keep in mind that your long hours of work are appreciated. However, the code review process is for improving the code you wrote, and not for criticizing it :blush: 

### **Required Changes** :recycle: 
Check the comments under the review.

### **Optional Changes** :zap: 
_Every comment with the **OPTIONAL** prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


* * *
**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear. You can mention me (@teekaytech) so I can be notified about your comments and respond._

*REMINDER: As described in the [Code reviews limit Policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).*

_**Please make the required changes and submit another review request when you are done.**_

* * *

_Reviewed by:_ [**Taofeek Olalere**](https://github.com/teekaytech)
__Cheers & Happy coding :smiley:__

**Review by Temmarie — APPROVED** [review]
## Status: Approved ✔️ 
### Hi @leonardodiasb , 
#### Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
#### Congratulations! 🎉 

#### Cheers and Happy coding!👏👏👏

#### Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
------
_As described in the [Code reviews limits policy](<https://microverse.zendesk.com/hc/en-us/articles/1500004088561>) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](<https://airtable.com/shrQAqnBwek5a0O0s>)._

**Pull request merged**